backend/backend/core.py

```diff
@@ -27,38 +24,20 @@ async def init(name: str, background_tasks: BackgroundTasks) -> bool:
     _class = params.pop("class")
     Alg = getattr(algs, _class)
     alg = Alg(n=config["n"], **params)
-    samplers[name] = alg
+
+    if hasattr(alg, "get_query"):
+        @app.get(f"/query-{name}")
+        def _get_query():
```

Comment:
Can this be configured so it's not running in `uvicorn` app?

Reply:
It's now running in a (single) gunicorn worker. That's close enough.

Future work is running these in multiple workers.

Reply:
Plus, there will be process monitoring soon with Dask.